### PR TITLE
Reduce PDS flow transaction polling interval

### PR DESCRIPTION
**Pull request by EricLin-wk**
- Reduce PDS flow transaction polling interval from 100ms to 3s.
- Make transaction polling interval configurable. Default 3s, local test default 1s.

### CODE REVIEW

File: e2e_test.go [hunk 1]
```diff
@@ -268,7 +268,7 @@ func TestE2E(t *testing.T) {
 			distribution = *d
 			break
 		}
-		time.Sleep(time.Second)
+		time.Sleep(cfg.TransactionPollInterval + time.Millisecond*100)
```
Comment (fkenji) [inline e2e_test.go]: Is `time.Millisecond*100` just an extra margin for waiting?

**Review by fkenji — APPROVED** [review]
This is great! 👍

File: e2e_test.go (hunk 1 above)
Comment (EricLin-wk) [reply e2e_test.go]: yep exactly

**Review by judezhu — APPROVED** [review]
LGTM!

**Pull request merged**